### DISCUSSION TITLE
Video does not capture Mouse pointer, am I missing anything?

### DISCUSSION THREAD

**Discussion by chilakamarthi** [baSSiLL/SharpAvi#53] [Q&A] [answered]
For me video recording works good but it does not capture mouse movements in Video. Any suggestion on fixing this issue?

**Comment by baSSiLL** [comment, marked as answer]
It's just a sample, so I didn't bother to make it a feature-rich screen casting utility. Although I've made some research on this topic back then and there is a comment in the code referring to this article
http://www.codeproject.com/Articles/12850/Capturing-the-Desktop-Screen-with-the-Mouse-Cursor
So take a look if you are interested.

**Reply by chilakamarthi** [reply]
I could change code and cursor is in recording now. Good pointer
